### DISCUSSION TITLE
Как используется GameMode.Script и для чего?

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#152] [Q&A]
**/// <summary>
	/// хранит значения параметров игрового режима
	/// </summary>
	[ScriptingRoot("GameMode")]
	public interface IGameModeService
	{
		/// <summary>
		/// скрипт игрового режима
		/// </summary>
		string Script { get; }**

**Comment by ZhenyaMartynov** [comment]
Скрипт игрового режима
Типо там Teams.Add итд..

**Reply by gat3v** [reply]
это знаю, но как и зачем его использовать то?

**Reply by ZhenyaMartynov** [reply]
GameMode.Script// получим скрипт сервера

**Reply by gat3v** [reply]
Понятно, но и зачем же тогда его получать?

**Reply by ZhenyaMartynov** [reply]
Как хочешь

**Comment by kkohno** [comment]
так, ну это нужно было когдато. возможно уберем... но вообще получить скрипт очевидно

**Comment by kkohno** [comment]
так, это вообще дыра в безопасности скриптов пооду)) убирем щас

**Comment by kkohno** [comment]
вырезали это уже кстати
